libsrc/core/utils.hpp

```diff
@@ -65,7 +67,7 @@ namespace ngcore
   {
 #if defined(__APPLE__) && defined(NETGEN_ARCH_ARM64)
     return mach_absolute_time();
-#elif defined(NETGEN_ARCH_AMD64)
+#elif defined(NETGEN_ARCH_AMD64) || defined(NGCORE_HAVE_RDTSC)
```

Comment:
change has no effect, NGCORE_HAVE_RDTSC is only ever defined if NETGEN_ARCH_AMD64 is defined.